Common/src/CConfig.cpp

```diff
@@ -4967,6 +4975,10 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
      }
    }
 
+  /*--- Check for wall catalycity. ---*/
+
+  if (nWall_Catalytic > 0) Catalytic = true;
```

Comment:
Remove the boolean from the class and use this check to implement GetCatalytic

Reply:
Im quite sure I understand.  
Are you saying I should I do something like:
`  /*!
   * \brief Determines if problem has catalytic walls.
   * \return true if catalytic walls are present.
   */
  bool GetCatalytic(void) const { if (nWall_Catalytic > 0) return true; }`

Reply:
`return nWall_Catalytic > 0`